### DISCUSSION TITLE
[Annoucement] Deprecating 'avoid_cache' property in Android

### DISCUSSION THREAD

**Discussion by simonzhexu** [WhatsApp/stickers#766] [Ideas]
Hi Folks,

We have had the following fields in Android:

`image_data_version` : an overall representation of the version of the stickers and tray icon. When you update stickers or tray icon in your pack, please update this string, this will tell WhatsApp that the pack has new content and update the stickers on WhatsApp side.

`avoid_cache` : this tells WhatsApp that the stickers from your pack should not be cached. By default, you should keep it false. Exception is that if your app updates stickers without user actions, you can keep it true, for example: your app provides clock sticker that updates stickers every minute.

Due to some internal development, we have decided to deprecate 'avoid_cache', If your app is using this field, please replace it with "image_data_version", by providing an updated "image_data_version", WhatsApp will pull your update. This field can be a string, so you can calculate a hash of your content, and provide an update, or you can treat that as integer, just increase it when you provide an update. 

If you have any questions, please post as a comment. 

Thanks.
Simon

**Comment by santi-petersen** [comment]
Should we remove it from the METADATA_CODE_FOR_SINGLE_PACK? Or can we keep it false?

**Reply by simonzhexu** [reply]
Please keep it false for now.

**Comment by osamz** [comment]
So, the problem is that users usually delete stickers app after adding packs, this option prevent this behavior. Now, what is the replacement?
Will all stickers be cached? or not?
